tap_wrike/streams.py

```diff
@@ -104,6 +104,14 @@ class dotdict(dict):
         "replication_method": "INCREMENTAL",
         "replication_key": "change_datetime",
     }),
+    "data_export_api_work_custom_field_history": dotdict({
+        "csv": True,
+        "path": "work_custom_field_history",
+        "params": {},
+        "key_properties": ["work_id", "user_resource_id", "old_custom_field_value", "custom_field_id", "change_datetime"],
```

Comment:
Are we sure this is the right unique index for this table? It seems a bit strange 🤔

Reply:
Good catch; we need to remove the 2 value columns

Reply:
@jeromepl what does the key_properties entry actually mean? Primary keys, or propoerties to be outputed?

Reply:
The primary key. In most cases it's just a single ID column, but sometimes its a more complex composite like it seems to be here

Reply:
Thanks! I was able to find the right values from [here](https://developers.wrike.com/schema-reference/)